CHANGES.rst

```diff
@@ -1751,6 +1751,8 @@ Bug Fixes
     raises a ``TypeError`` on them (while still allowing ``Time`` arrays to be
     iterated). [#4048]
 
+  - Added a new example of a custom time format class ``ydate_custom``. [#4703]
```

Comment:
I would say that this PR does **not** need a change log, given that it is only adding stuff to the doc.